app/jobs/elasticsearch_job.rb

```diff
@@ -0,0 +1,29 @@
+class ElasticsearchJob < ActiveJob::Base
+    queue_as :default
+
+    rescue_from ActiveJob::DeserializationError, Faraday::TimeoutError do
+        retry_job wait: 5.minutes, queue: :default
+    end
+
+    def perform(data, operation)
+      options = { content_type: 'application/vnd.api+json', accept: 'application/vnd.api+json', bearer: ENV['JWT_TOKEN'] }
```

Comment:
I think this is the only place where we use a JWT token. How do we generate the token?